### PR TITLE
fix(connectToggle): use currentRefinement in metadata instead of the label

### LINKED ISSUE #891: connectToggle.js properties docs
```
@providedPropType {boolean} currentRefinement - the refinement currently applied
```

should be replaced with something like

```
@providedPropType {boolean} currentRefinement - current checkbox value
```

because it is not "the refinement currently applied"

### PR DESCRIPTION
**Pull request by samouss**
**Summary**

Close partially #891

### CODE REVIEW

File: packages/react-instantsearch/src/connectors/connectToggle.js [hunk 1]
```diff
@@ -109,14 +109,16 @@ export default createConnector({
     const checked = getCurrentRefinement(props, searchState, this.context);
     const items = [];
     const index = getIndex(this.context);
+
     if (checked) {
       items.push({
```
Comment (Haroenv) [inline packages/react-instantsearch/src/connectors/connectToggle.js]: really weird conditional pushing IMO

File: packages/react-instantsearch/src/connectors/connectToggle.js [hunk 2]
```diff
@@ -109,14 +109,16 @@ export default createConnector({
...
         label: props.label,
-        currentRefinement: props.label,
+        currentRefinement: checked,
```
Comment (Haroenv) [inline packages/react-instantsearch/src/connectors/connectToggle.js]: makes sense wrt that issue

File: packages/react-instantsearch/src/connectors/connectToggle.js (hunk 1 above)
Comment (samouss) [reply packages/react-instantsearch/src/connectors/connectToggle.js]: Which behaviour you will expect instead?

Comment (Haroenv) [reply packages/react-instantsearch/src/connectors/connectToggle.js]:
```js
const items = checked ? [{..stuff}] : []
```

I'm not sure if it's that much better though

Comment (samouss) [reply packages/react-instantsearch/src/connectors/connectToggle.js]: Yep not sure, because the ternary will be split on several line.

**Pull request merged**